### ISSUE TITLE
TeeReader

### ISSUE THREAD

**Issue by yegor256** [yegor256/cactoos#397]
Would be great to create `TeeReader`, similar to `TeeInputStream` we already have.

**Comment by memoyil** [comment]
Ok i'll do it

**Comment by memoyil** [comment]
@yegor256 I think we should close this issue

**Comment by yegor256** [comment]
implemented in #398

**Comment by 0crat** [comment]
I can't do it for technical reasons, I'm very sorry. If you don't know what to do, submit this error as a ticket [here](https://github.com/zerocracy/datum):

```
1.0-SNAPSHOT bb35857 2017-08-15T16:58:39
IllegalStateException: java.lang.reflect.InvocationTargetException
com.zerocracy.jstk.SoftException: Job `gh:yegor256/cactoos#397` was not in scope
	at com.zerocracy.pm.scope.Wbs.remove(Wbs.java:107)
	at com.zerocracy.pm.scope.Wbs$remove$1.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at com.zerocracy.stk.pm.scope.wbs.remove_job_from_wbs.exec(remove_job_from_wbs.groovy:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:109)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:90)
	at com.zerocracy.farm.reac...
```
